servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DefaultDnsServiceDiscovererBuilder.java

```diff
@@ -44,10 +49,44 @@
  */
 @Deprecated // FIXME: 0.43 - make package private
 public final class DefaultDnsServiceDiscovererBuilder implements DnsServiceDiscovererBuilder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DefaultDnsServiceDiscovererBuilder.class);
+
+    /**
+     * @deprecated This system property is introduced temporarily as a way for users to skip binding and revert the
+     * preexisting behavior.
+     */
+    @Deprecated // FIXME: 0.43 - consider removing this system property
+    private static final String SKIP_BINDING_PROPERTY = "io.servicetalk.dns.discovery.netty.skipBinding";
+    @Nullable
+    private static final SocketAddress DEFAULT_LOCAL_ADDRESS =
+            getBoolean(SKIP_BINDING_PROPERTY) ? null : new InetSocketAddress(0);
```

Comment:
For reference, `java.net.DatagramSocket#send(DatagramPacket)` uses `new InetSocketAddress(0)` if it's not bound yet.